challenge3/webapp/goi/app/controllers/api/v1/dashboard_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::V1::DashboardController < ApplicationController
+  def index
+    energies = House.city_energy_production
+    energy_production = []
+    date = []
+    cambridge = []
+    london = []
+    oxford = []
+
+    energies.each.with_index {|(k, v), i|
```

Comment:
```suggestion
    energies.each.with_index {|((year, month), v), i|
```
の方が良いのではないか。

好みの問題かもしれないけれど。

Reply:
悪い癖です。ループカウンタ以外は省略しないよう気をつけます。